modules/zapier/class-module-zapier.php

```diff
@@ -79,13 +79,32 @@ public function pull_the_trigger( array $data, $hook_url ) {
              *
              * @since    1.1.0
              */
-            $result = wp_remote_post( $hook_url, apply_filters( 'ctz_post_request_args', $args ) );
-
-            // If result is a WP Error, throw a Exception woth the message.
-            if ( is_wp_error( $result ) ) {
```

Comment:
don't removal `is_wp_error( $result )` broke some cases?

Reply:
Since it's not working with our API it more misleading than helping